### DISCUSSION TITLE
Trailingslash  to configure： I want to realize that some routes need slashes, and some routes don't need slashes

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#34438] [Ideas]
### Describe the feature you'd like to request

such as： http://localhost:3000/zhaohuo

http://localhost:3000/zhaohuo/a1b2/

### Describe the solution you'd like

no

### Describe alternatives you've considered

no

**Comment by parlay96** [comment]
Trailingslash to configure： I want to realize that some routes need slashes, and some routes don't need slashes

**Comment by parlay96** [comment]
Can anyone see it？thank you

**Comment by swcarter007** [comment]
I would also like to see the trailing slash be optional on urls.  I'm trying to use nextjs in a site that contains both trailing slash urls and non-trailing slash urls.  The ones with the trailing slash indicate it has child elements while the non-trailing slash indicates its a leaf url.  There should be an option to default urls to a certain standard, while still maintaining the ability to override that standard on a per url basis without doing a 308 without having to configure a custom server to handle the requests.
